http.md

```diff
@@ -0,0 +1,342 @@
+# JSON RPC HTTP Methods
+
+searcher nodes accept HTTP requests using the [JSON-RPC 2.0](https://www.jsonrpc.org/specification) specification.
+
+## RPC HTTP Endpoint
+
+**Default port:** 8899 e.g. [http://localhost:8899](http://localhost:8899), [http://192.168.1.88:8899](http://192.168.1.88:8899)
+
+## Request Formatting
+
+To make a JSON-RPC request, send an HTTP POST request with a `Content-Type: application/json` header. The JSON request data should contain 4 fields:
```

Comment:
maybe just link to solana docs instead? https://docs.solana.com/api/http#request-formatting